tasks/task01/Narcissistic_Number.py

```diff
@@ -0,0 +1,16 @@
+for k in range(1, 1000) :
+    if(k>=1 and k<10):
+        digit=k%10
+        if digit**1==k:
+         print ('Narcissistic__Number', k)
+    if(k>=10 and k<100):
+            digit1=k//10
+            digit2= k%10
+            if digit1**2+digit2**2==k:
+                print ('Narcissistic__Number', k)
+    if (k>=100 and k<1000):
+                    digit3=k%10
+                    digit4= k%100//10
+                    digit5= k//100
+                    if digit5**3+digit4**3+digit3**3==k:
+                        print('Narcissistic__Number', k)
```

Comment:
```suggestion
                        print('Narcissistic__Number', k)

```

по PEP8 код должен заканчиваться пустой строкой

Примени мои коментарии ко всему коду.